src/boot/boot.js

```diff
@@ -163,6 +185,10 @@ export function bootHypothesisClient(doc, config) {
  * @param {SidebarAppConfig} config
  */
 export function bootSidebarApp(doc, config) {
+  // Preload `/api/` and `/api/links` API responses.
+  preloadUrl(doc, 'fetch', config.apiUrl);
+  preloadUrl(doc, 'fetch', config.apiUrl + 'links');
```

Comment:
```suggestion
  preloadUrl(doc, 'fetch', `${config.apiUrl}links`);
```

Reply:
We don't have a firm convention about when to use string interpolation vs. concatenation in our code. I think the informal approach up to this point has been "use interpolation when it makes obvious sense, concatenation is fine for simple cases (like adding a prefix or suffix)". Happy to discuss more in Slack.